build.rs

```diff
@@ -13,27 +13,26 @@ fn main() {
 
     cmd.current_dir(Path::new("rocksdb"))
         .arg("EXTRA_CFLAGS=-fPIC")
-        .arg("EXTRA_CXXFLAGS=-fPIC")
+        .arg("EXTRA_CXXFLAGS=-fPIC") // -std=c++11 -stdlib=libstdc++
         .arg(format!("INSTALL_PATH={}", out_dir));
 
     if let Ok(jobs) = num_jobs {
         cmd.arg(format!("-j{}", jobs));
     }
 
-    cmd.arg("install-static");
+    cmd.arg("install");
```

Comment:
I've been told `rustc` uses `ld` as the linker, in fact, but not 100% sure. Anyway, I got lots of "undefined symbol" errors if I tried to link it statically.